python/nano/src/bigdl/nano/tf/quantization.py

```diff
@@ -21,6 +21,7 @@
 
 
 def quantize(self,
+             precision='int8',
```

Comment:
will it be possible to support other accelerators in future (such as openvino)?

Reply:
Yes, sure for openvino. As for onnx, maybe we can follow https://onnxruntime.ai/docs/tutorials/tf-get-started.html. But before that, we should figure out how to wrap a quantized tf model.

Reply:
The we may need an `accelerator` parameter?